dnf/cli/commands/__init__.py

```diff
@@ -232,7 +232,7 @@ class ProvidesCommand(Command):
     """
 
     aliases = ('provides', 'whatprovides', 'prov', 'wp')
-    summary = _('find what package provides the given value')
+    summary = _('find what package provides the given file.')
```

Comment:
Please also drop this change, like I said before this command is not specific to file provides.

Reply:
Ok.